packages/experimental/Shimmer/src/Shimmer.types.ts

```diff
@@ -0,0 +1,135 @@
+import { SvgProps, RectProps, CircleProps, ImageProps } from 'react-native-svg';
+import { ImageURISource } from 'react-native';
+
+export const shimmerName = 'Shimmer';
+/**
+ * Specifies the possible type of the shimmer elements.
+ * Required when building more than one element at a time.
+ */
+export enum ShimmerElementType {
+  /**
+   * Rect element type
+   */
+  rect = 1,
+  /**
+   * Circle element type
+   */
+  circle = 2,
+}
+/**
+ * Represents the shimmer element
+ */
+export interface ShimmerElement {
+  /**
+   * Type of the shimmer element
+   */
+  type?: ShimmerElementType;
+  /**
+   * Specifies the element width, non-op for circle element
```

Comment:
width in points? pixels?

Reply:
In dp, if clients want they can use PixelRatio to get the device pixel density and calculate the dimensions based on that?

Reply:
I was more being nitpicky about having units in documentation, but also because I truly did not know

Reply:
I think dp is the default 'unit' in react native, not sure if it's redundant to specify that in doc